core/variant/variant.cpp

```diff
@@ -2832,7 +2832,7 @@ uint32_t Variant::hash() const {
 			return _data._bool ? 1 : 0;
 		} break;
 		case INT: {
-			return _data._int;
+			return (uint32_t)hash_one_uint64(_data._int);
```

Comment:
What I meant is that the argument would need a cast, since it's an `int64_t` buy `hash_one_uint64()` expect an `uint64_t`. The function already returns a `uint32_t` so it won't need a cast in the return value.

Reply:
Resolved